tests/integration/targets/dcnm_interface/tests/dcnm/dcnm_lo_mpls.yaml

```diff
@@ -0,0 +1,151 @@
+##############################################
+##               SETUP                      ##
+##############################################
+
+# This test file specfically validates fabric loopback
+# interfaces that are created at fabric creation time
+# for VXLAN-EVPN to SR-MPLS and MPLS LDP interconnection
+
+# By default they are loopback101 as defined in the fabric
+# settings and use the ndfc profile 'int_mpls_loopback'
+
+- name: Remove local log file
+  local_action: command rm -f dcnm_intf.log
+
+- name: Save Data For Loopback101
+  cisco.dcnm.dcnm_interface:
+    check_deploy: True
+    fabric: "{{ ansible_it_fabric }}"
+    state: query
+    config:
+      - name: lo101
+        switch:
+          - "{{ ansible_switch1 }}"
+  register: lpk101_data
+
+- name: Set Test Control Flag
+  ansible.builtin.set_fact:
+    run_test: True
+
+- name: Test will not be run since Loopback101 does not appear to exist
+  ansible.builtin.set_fact:
+    run_test: False
+  when: lpk101_data.response == []
+
+- name: Access Policy Data If Loopack101 Data Is Available
+  ansible.builtin.set_fact:
+    loopback_policy: "{{ lpk101_data.response[0]['policy'] }}"
+  when: lpk101_data.response != []
+
+- name: Test will not be run since Loopback101 is not an MPLS Fabric Loopback
+  ansible.builtin.set_fact:
+    run_test: False
+  when: (loopback_policy is defined) and (loopback_policy != 'int_mpls_loopback')
+
+- debug: msg="Loopback 101 Data - {{ lpk101_data.response[0]['interfaces'][0]['nvPairs'] }}"
+  when: run_test
+
+- block:
+
+##############################################
+##                MERGE                     ##
+##############################################
+
+    - name: Modify mpls fabric loopback interface
+      cisco.dcnm.dcnm_interface: &lo_merge
+        check_deploy: True
+        fabric: "{{ ansible_it_fabric }}"
+        state: merged
```

Comment:
How about state replaced? Can we replace an existing fabric loopback interface with the same type but different parameters